TODO.md

```diff
@@ -0,0 +1,28 @@
+Systers Portal TODO 
+===================
+
+This file is that medium that is used to keep you updated on the exiciting features we are working on or what we are bringing to the Systers Portal next! 
```

Comment:
We keep you updated on the features we are currently working on or planning to work in the future and are not still available in the current release. In case you have request for a new feature, please submit a git issue and add [Feature Suggestion] before the title.